### DISCUSSION TITLE
Overflow Scroll + Flex Grow

### DISCUSSION THREAD

**Discussion by djnugent** [tailwindlabs/tailwindcss#11742] [Help] [answered]
I cant for the life of me figure out how to make `col1` scroll. You can set height of `trifold` to  `h-full` but then the page no longer fits in the window (because it would be header + 100% aka header + 100vh) and there is a tiny page scroll that is the height of the `<h1>`.

[Version with grow:](https://play.tailwindcss.com/2QcL96MYwV) overflow doesnt work
[Version with h-full:](https://play.tailwindcss.com/fhfDW9CkYb) overflow works but now the page doesnt fit inside window
- The effect is minor in this toy example but more major in my actual application

Any thoughts?

**Comment by djnugent** [comment, marked as answer]
I came to a solution. Using absolute positioning to grow my divs
https://play.tailwindcss.com/oL097n5qf0

**Comment by kirkbushell** [comment]
Just FYI there's a way to do it without absolute positioning, if you're still interested :)
